v2/tools/generator/internal/config/configuration.go

```diff
@@ -46,8 +46,6 @@ type Configuration struct {
 	// AnyTypePackages lists packages which we expect to generate
 	// interface{} fields.
 	AnyTypePackages []string `yaml:"anyTypePackages"`
-	// Filters used to control which types are exported
-	ExportFilters []*ExportFilter `yaml:"exportFilters"`
```

Comment:
Oh, you've removed this and the YAML has this section still (I think you forgot to remove it), but there's no error when that happens?

Can we pass a "error on unknown fields" to the YAML deserializer?

Reply:
Happy to deal with this in a separate PR though

Reply:
Created issue #2081 to track.